src/content/docs/apm/agents/php-agent/other-integrations/magento-business-insights.mdx

```diff
@@ -0,0 +1,263 @@
+    
+    <Step>
+        ## Install the Apache integration
+
+        Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
+
+        To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
+        
+If you need to install the agent in a different way, see [Apache monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration).
+
+        </Step>
+    
+
+    <Step>
+        ## Install the Elasticsearch integration
+        
+        Our Elasticsearch integration collects metrics like cluster status and current shards. If you're using [Magento version 2.4 and higher](https://magentip.com/install-magento-2-4-x-on-ubuntu-with-elasticsearch/), you'll to install an ElasticSearch application and agent to monitor that application. 
+        
+        To install the integration:
+
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add data**.
+        2. Search for and click on **Elasticsearch**.
+        3. Select an account and follow the guided install steps.
+
+If you need to install the agent in a different way, see [Apache monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration).
+
+        </Step>
+    
+
+    <Step>
+        ## Set up synthetic monitoring
+
+        Our synthetic monitors test your app in different locations and capture server downtime. You can view this data in the Magento Business Insights dashboard.
+
+        To set up synthetic monitors, follow the steps in our [guided install](https://onenr.io/0EjOXrnxgQ6).
+
+        </Step>
+    
+
+    <Step>
+        ## Install the AWS Identity and Access Management (IAM) integration
+        
+        Our AWS IAM integration collects metrics such as certificates per account, groups per account, MFA devices per account, roles per account, and users per account.
+
+        To install the integration:
+
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/) > Add data**.
+        2. Search for and click on **Integrate your AWS account**. 
+        2. Select an account and follow the guided install steps.
+
+For more information on this integration, see [AWS IAM integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-iam-monitoring-integration)
+        </Step>
+    
+
+    <Step>
+        ## Install the Flex integration
+
+        Our Flex integration allows you to capture business metrics through queries of your app database.
```

Comment:
we might want to explain that the Flex integration isn't really a specific integration but something that allows you to make a custom integration/data-collection of your own. If I didn't know about it, I'd be a little confused thinking it was some specific kind of integration, not just a custom thing you set up.